File: week-02/day-01/cuboid.py
# Write a program that stores 3 sides of a cuboid as variables (doubles)
# The program should write the surface area and volume of the cuboid like:
# 
# Surface Area: 600
# Volume: 1000

length = int(input('Length: '))
width = int(input('width: '))
height = int(input('height: '))

surface = (length + width + height) * 2
volume = length * width * height

print('surface area:', surface)
print('volume: ', volume)

File: week-07/day-01/cuboid.js
'use strict';

// Write a program that stores 3 sides of a cuboid as variables (floats)
// The program should write the surface area and volume of the cuboid like:
//
// Surface Area: 600
// Volume: 1000

var length = 100;
var height = 200;
var width = 300;

var surface = (length + width + height) * 2;
var volume = length * width * height;

console.log('Surface: ' + surface);
console.log('Volume: ' + volume);
